submissions/VolodymyrRutskyi/task_js_dom/main.js

```diff
@@ -0,0 +1,186 @@
+            если элемент удовлетворяет условию проверяющей 
+            функции. В противном случае возвращается -1.`
+        ]
+    },
+    {
+        name: 'forEach',
+        image: 'img/foreach.jpg',
+        description: [
+            `Метод forEach() выполняет указанную функцию
+             один раз для каждого элемента в массиве.`
+        ]
+    },
+    {
+        name: 'Includes',
+        image: 'img/includes.jpg',
+        description: [
+            `Метод includes() определяет, содержит
+            ли массив определённый элемент, возвращая в 
+            зависимости от этого true или false.`
+        ]
+    },
+    {
+        name: 'Join',
+        image: 'img/join.jpg',
+        description: [
+            `Метод join() объединяет все элементы 
+            массива (или массивоподобного объекта) в строку.`
+        ]
+    },
+    {
+        name: 'Map',
+        image: 'img/map.jpg',
+        description: [
+            `Метод map() создаёт новый массив с результатом 
+            вызова указанной функции для каждого элемента массива.`,
+            `Метод map() не изменяет данный массив, создает копию.`
+        ]
+    },
+    {
+        name: 'Push',
+        image: 'img/push.jpg',
+        description: [
+            `Метод push() добавляет один или более элементов 
+            в конец массива и возвращает новую длину массива.`
+        ]
+    },
+    {
+        name: 'Reverse',
+        image: 'img/reverse.jpg',
+        description: [
+            `Метод reverse() на месте обращает порядок следования
+            элементов массива. Первый элемент массива становится
+            последним, а последний — первым.`
+        ]
+    },
+    {
+        name: 'Slice',
+        image: 'img/slice.jpg',
+        description: [
+            `Метод slice() возвращает новый массив, 
+            содержащий копию части исходного массива.`,
+            `Метод slice() не изменяет данный массив, создает копию.`
+        ]
+    },
+    {
+        name: 'Some',
+        image: 'img/some.jpg',
+        description: [
+            `Метод some() проверяет, удовлетворяет ли 
+            какой-либо элемент массива условию, заданному 
+            в передаваемой функции.`
+        ]
+    }
+];
+
+const main = document.querySelector('.main');
+const navList = document.querySelector('.nav__list');
+const content = document.querySelector('.content');
+
+// function contentDisplay() {
+//     dataList.map((item) =>{
+//         navList.innerHTML += `<li class='nav__item'>${item.name}</li>`;
+//     })
+//     main.prepend(navList);
+// }
+
+function contentDisplay() {
+    const updatedList = dataList.map((item) =>{
+        return `<li class='nav__item'>${item.name}</li>`;
+    })
+    navList.innerHTML = updatedList.join('');
+    main.prepend(navList);
+}
+
+function buildContent(event){
+    let btns= document.querySelectorAll('.nav__item');
+    dataList.map((item) => {
+        if (event.target.innerHTML === item.name) { 
```

Comment:
```suggestion
        if (event.target.innerText === item.name) { 
```

Reply:
If i use innerText, buildContent function isn't working. I can use innerHTML or textContent

Reply:
`<li>Create <b>project</b></li>`
innerHTML get all from tag `Create <b>project</b>`
innerText get an only the text `Create project`